src/metabase/email/login_from_new_device.mustache

```diff
@@ -0,0 +1,23 @@
+{{> metabase/email/_header }}
+<div style="margin: 2em 4em; border: 1px solid #dddddd; border-radius: 2px; background-color: white; box-shadow: 0 1px 2px rgba(0, 0, 0, .08); text-align: center;">
+  <h2 style="font-weight: normal; color: #4C545B;line-height: 1.65rem;">
+    We've Noticed a New Login, {{first-name}}
+  </h2>
+  <p style="margin-top: 1em">
+    We noticed a login from a device you don't usually use.
+  </p>
```

Comment:
I don't see any mention of this being about the user's Metabase account. We should make it clearer what this email is about.

```suggestion
  <h2 style="font-weight: normal; color: #4C545B;line-height: 1.65rem;">
    We've noticed a new login on your Metabase account.
  </h2>
  <p style="margin-top: 1em">
    Hi {{first-name}},
  </p>
  <p style="margin-top: 1em">
    We noticed a login on your Metabase account from a new device.
  </p>
```

Perhaps we should similarly update the subject:

> Metabase account login from new device

This was just off the top of my head. I'm sure @kdoh or @mazameli could do better.